crates/rolldown_common/src/types/module_info.rs

```diff
@@ -12,3 +12,17 @@ pub struct ModuleInfo {
   pub imported_ids: Vec<ModuleId>,
   pub dynamically_imported_ids: Vec<ModuleId>,
 }
+
+impl From<ModuleId> for ModuleInfo {
```

Comment:
I'll suggest using `ModuleInfo::new`.

Reply:
Here `ModuleInfo::new` needs to care about more fields, the external module only has `id`.

Reply:
It's for clear reading

```diff
- self.shared_context.plugin_driver.set_module_info(&id.clone(), Arc::new(id.into()));
+ self.shared_context.plugin_driver.set_module_info(&id.clone(), Arc::new(moduleInfo::new(id)));
```

Reply:
> Here `ModuleInfo::new` needs to care about more fields, the external module only has `id`.

The `MouleInfo::new` does not follow the rust struct style, let using struct litreal at here.